javascript/computer_science/project_knights_travails.md

```diff
@@ -1,41 +1,43 @@
 ### Introduction
 
-Now you're a pro with DFS and BFS.  Let's try using our search algorithms on a real problem.
+Now that you're pro with DFS and BFS. Let's try using our search algorithms on a real problem.
 
 For this project, you'll need to use a data structure that's similar (but not identical) to a binary tree. For a summary of a few different examples, reference [this article](https://www.khanacademy.org/computing/computer-science/algorithms/graph-representation/a/describing-graphs).
 
-A knight in chess can move to any square on the standard 8x8 chess board from any other square on the board, given enough turns (don't believe it?  See [this animation](https://cdn.statically.io/gh/TheOdinProject/curriculum/284f0cdc998be7e4751e29e8458323ad5d320303/ruby_programming/computer_science/project_knights_travails/imgs/00.png)).  Its basic move is two steps forward and one step to the side.  It can face any direction.
+A knight in chess can move to any square on the standard 8x8 chess board from any other square on the board, given enough turns (don't believe it? See [this animation](https://cdn.statically.io/gh/TheOdinProject/curriculum/284f0cdc998be7e4751e29e8458323ad5d320303/ruby_programming/computer_science/project_knights_travails/imgs/00.png)). Its basic move is two steps forward and one step to the side or one step forward and two steps to the side. It can face any direction.
 
-All the possible places you can end up after one move look like this:
+All the possible places you can end up after one move looks like this:
 
 ![Knights Travails board](https://cdn.statically.io/gh/TheOdinProject/curriculum/d30038e0aaca1f35e58e205e37a21b2c9d31053d/javascript/computer_science/project_knights_travails/imgs/01.png)
 Note: The picture is only to explain the problem, There is no need to create a GUI.
 
 ### Assignment
+
 Your task is to build a function `knightMoves` that shows the shortest possible way to get from one square to another by outputting all squares the knight will stop on along the way.
 
-You can think of the board as having 2-dimensional coordinates.  Your function would therefore look like:
+You can think of the board as having 2-dimensional coordinates. Your function would therefore look like:
 
-  * `knightMoves([0,0],[1,2]) == [[0,0],[1,2]]`
+- `knightMoves([0,0],[1,2]) == [[0,0],[1,2]]`
 
-  Note: Sometimes _there is more than one fastest path_, an example of this is shown below. Either answer will work.
+Note: Sometimes _there are more than one fastest path_, an example of this is shown below. Either answer will work.
```

Comment:
```suggestion
Note: Sometimes _there is more than one fastest path_, an example of this is shown below. Either answer will work.
```
Again the old way is correct. The subject is the singular noun phrase "one fastest path", so the verb should be the singular "is".